java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -3356,6 +3357,21 @@ public <ArgType> T xadd(@NonNull ArgType key, @NonNull Map<ArgType, ArgType> val
         return xadd(key, values, StreamAddOptions.builder().build());
     }
 
+    /**
+     * Adds an entry to the specified stream stored at <code>key</code>.<br>
+     * If the <code>key</code> doesn't exist, the stream is created.
+     *
+     * @implNote {@link ArgType} is limited to {@link String} or {@link GlideString}, any other type
+     *     will throw {@link IllegalArgumentException}.
+     * @see <a href="https://valkey.io/commands/xadd/">valkey.io</a> for details.
+     * @param key The key of the stream.
+     * @param values Field-value pairs to be added to the entry.
+     * @return Command Response - The id of the added entry.
+     */
+    public <ArgType> T xadd(@NonNull ArgType key, @NonNull ArgType[][] values) {
```

Comment:
Please add UT for new methods or update existing tests

Reply:
Which UTs need to be added/updated? Just need to be pointed in the right direction.